django-stubs/contrib/admin/helpers.pyi

```diff
@@ -146,7 +147,7 @@ class InlineAdminFormSet:
     def inline_formset_data(self) -> str: ...
     @property
     def forms(self) -> list[BaseForm]: ...
-    @property
+    @cached_property
     def non_form_errors(self) -> Callable[[], ErrorList]: ...
```

Comment:
This looks like a regular method in Django 4.2. Not sure why it's typehinted so weird. Am I missing something?

Reply:
Ye that is weird, it's a regular method your right. I'll fix that